launch/tier4_planning_launch/launch/scenario_planning/parking.launch.py

```diff
@@ -62,7 +62,7 @@ def launch_setup(context, *args, **kwargs):
                         "vehicle_frame": "base_link",
                         "map_frame": "map",
                         "update_rate": 10.0,
-                        "activate_by_scenario": False,
+                        "activate_by_scenario": True,
```

Comment:
@yuntianyi-chen Is this your intentionally change?

Reply:
Oh, sorry. It already discussed above.

Reply:
I didn't intend to change it initially. Ambroise suggested changing it in the [discussion ](https://github.com/autowarefoundation/autoware.universe/pull/4695#discussion_r1310449195)above.